src/api/queries/useGetUsersFromSupabase.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from "@tanstack/react-query"
+
+import { QUERY_KEYS } from "../../constants/query_keys"
+import { supabase } from "../../database/supabase"
+
+
+  export const useGetUsersFromSupabase = () => {
+    const {data:users, error, isLoading, refetch} = useQuery({queryKey:[QUERY_KEYS.getUsers], queryFn: async ()=> await supabase.from('users').select('*'), gcTime: 0})
+    
+    if (users && users.data) {
+        const data = users.data
+        return{data, isLoading, error}
+        }
+        return{data:[], isLoading, error, refetch}
+  }
```

Comment:
to chyba już nigdzie po tych naszych dzisiejszych zmianach nie jest używane więc można wywalić

Reply:
używane jest w statistics provider